pegasus/emails/hoc_signup_2016_receipt.md

```diff
@@ -0,0 +1,40 @@
+---
+from: "Hadi Partovi (Code.org) <hadi_partovi@code.org>"
+subject: "Thanks for signing up to host an Hour of Code!"
+litmus_tracking_id: "5g5lyi1a"
+---
+<% hostname = CDO.canonical_hostname('hourofcode.com') %>
+
+# Thanks for signing up to host an Hour of Code!
+
+You're making it possible for students all around the world to learn one Hour of Code that can change the rest of their lives, during <%= campaign_date('campaign_date_short') %>.
```

Comment:
Same as 2015 email, but removed prizes info. I can ask Jon to review once it's on staging.